pulp_container/app/models.py

```diff
@@ -200,6 +200,50 @@ class Meta:
         unique_together = (("name", "tagged_manifest"),)
 
 
+class ManifestSignature(Content):
+    """
+    A signature for a manifest.
+
+    Fields:
+        name (models.CharField): A signature name in the 'manifest_digest@random_name' format
+        digest (models.CharField): A signature sha256 digest.
+        type (models.CharField): A signature type as specified in signature metadata. Currently
+                                 it's only "atomic container signature".
+        key_id (models.CharField): A key id identified by gpg (last 8 bytes of the fingerprint).
+        timestamp (models.PositiveIntegerField): A signature timestamp identified by gpg.
+        creator (models.CharField): A signature creator.
+        data (models.BinaryField): A signature, base64 encoded.
+
+    Relations:
+        signed_manifest (models.ForeignKey): A manifest this signature is relevant to.
+
+    """
+
+    PROTECTED_FROM_RECLAIM = False
+
+    TYPE = "signature"
+
+    SIGNATURE_CHOICES = ((SIGNATURE_TYPE.ATOMIC_SHORT, SIGNATURE_TYPE.ATOMIC_SHORT),)
+
+    name = models.CharField(max_length=255, db_index=True)
+    digest = models.CharField(max_length=255)
```

Comment:
maybe worth adding db_index

Reply:
it's indexed, it's in the uniqueness constraint below, and on its own.